src/test/streams_tests.cpp

```diff
@@ -399,6 +448,17 @@ BOOST_AUTO_TEST_CASE(streams_buffered_file_rand)
                 break;
             }
             case 3: {
+                // SkipTo is similar to the "read" cases above, except
+                // we don't receive the data.
+                size_t nSize = InsecureRandRange(5);
```

Comment:
nit: could use list initialization?

```suggestion
                size_t nSize{InsecureRandRange(5)};
```